src/api/.rubocop.yml

```diff
@@ -92,6 +92,9 @@ Lint/AssignmentInCondition:
 Lint/EndAlignment:
   AlignWith: variable
 
+Lint/Eval:
```

Comment:
Could you add a comment up to this line explaining what the cop does?. I think those comments may be useful. Something like:

`# checks for the use of *Kernel#eval* as it is a serious security risk.`